### PR TITLE
CIS report changes and add CIS 1.6

### PR DESCRIPTION
**Pull request by mrajashree**
This PR adds the CRs for CIS 1.6 benchmark and profiles. https://github.com/rancher/rancher/issues/29649
It also adds following changes to the clusterScan CRD:
1. AdditionalPrinterColumn "Warn" to show number of tests with "Warn" output. https://github.com/rancher/security-scan/issues/32
2. Field called `scoreWarning`, with which users can decide whether a scan should pass or fail if there are tests with "warn" output. https://github.com/rancher/cis-operator/issues/54

### CODE REVIEW

File: packages/rancher-cis-benchmark/charts/values.yaml
Comment (mrajashree) [inline packages/rancher-cis-benchmark/charts/values.yaml:8]: This will be changed once the final image is out, might even be removed from this PR

File: packages/rancher-cis-benchmark/charts/crds/clusterscan.yaml
Comment (mrajashree) [inline packages/rancher-cis-benchmark/charts/crds/clusterscan.yaml:22]: clusterScan CRD is also changed in the other PR to include recurring scans. So I'm not sure how to merge the PRs in which order, or will there be one PR to include all these changes?
@prachidamle @cbron

File: packages/rancher-cis-benchmark/charts/values.yaml
Comment (mrajashree) [reply packages/rancher-cis-benchmark/charts/values.yaml]: removing the image changes, will be done through @briandowns 's PR

**Review by prachidamle — COMMENTED** [review]
LGTM - let's merge after Brian's rke2 PR merges

**Review by prachidamle — APPROVED** [review]
LGTM

**Review by prachidamle — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**
